c-scape/src/math/mod.rs

```diff
@@ -728,6 +728,47 @@ unsafe extern "C" fn nearbyintf(x: f32) -> f32 {
     libm::rintf(x)
 }
 
+#[no_mangle]
+unsafe extern "C" fn finite(x: f64) -> i32 {
+    return x.is_finite() as i32;
+}
+#[no_mangle]
+unsafe extern "C" fn finitef(x: f32) -> i32 {
+    return x.is_finite() as i32;
+}
+
+#[no_mangle]
+unsafe extern "C" fn isnan(x: f64) -> i32 {
+    return x.is_nan() as i32;
+}
+
+#[no_mangle]
+unsafe extern "C" fn isnanf(x: f32) -> i32 {
+    return x.is_nan() as i32;
+}
+
+#[no_mangle]
+unsafe extern "C" fn isinf(x: f64) -> i32 {
+    if x == f64::INFINITY {
+        return 1;
+    } else if x == f64::NEG_INFINITY {
+        return -1;
+    } else {
+        return 0;
+    }
+}
+
+#[no_mangle]
+unsafe extern "C" fn isinff(x: f32) -> i32 {
+    if x == f32::INFINITY {
+        return 1;
+    } else if x == f32::NEG_INFINITY {
+        return -1;
+    } else {
+        return 0;
```

Comment:
In functions liike these `is*` functions, the style in c-ward and related repos is to omit the `return` keywords.

Reply:
is that specifically for returning known values (ie implicit return for 1) or all these bool like functions (so also for x.is_nan()) ?